frontend/src/pages/ForgotPassword.vue

```diff
@@ -13,7 +13,7 @@
         </div>
       </b-container>
     </div>
-    <b-container class="mt--8 p-1">
+    <b-container v-if="!showPageMessage" class="mt--8 p-1">
```

Comment:
v-if="!showPageMessage"  / v-else

I would change, so that there is no need for a negative query.

Reply:
But I like to have the most important tag first. 😁

To negate something is not negative. Or? 😉
If the expression gets to difficult to understand – to confising – then we should avoid those, I guess.